crawlerr/config.py

```
@@ -14,72 +14,79 @@ class CrawlerrSettings(BaseSettings):
+    server_port: int = Field(default=8000, alias="SERVER_PORT")
+    debug: bool = Field(default=False, alias="DEBUG")
+    production: bool = Field(default=False, alias="PRODUCTION")
     
     # Logging Configuration
-    log_level: str = Field(default="INFO", env="LOG_LEVEL")
-    log_format: str = Field(default="console", env="LOG_FORMAT")
-    log_file: Optional[str] = Field(default=None, env="LOG_FILE")
-    log_to_file: bool = Field(default=False, env="LOG_TO_FILE")
+    log_level: str = Field(default="INFO", alias="LOG_LEVEL")
+    log_format: str = Field(default="console", alias="LOG_FORMAT")
+    log_file: Optional[str] = Field(default=None, alias="LOG_FILE")
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)